src/icons/clarity-icons.scss

```diff
@@ -50,6 +51,9 @@ clr-icon {
     &.icon-color-success {
         @include setIconColor($clr-icon-color-success);
     }
+    &.icon-color-error {
+        @include setIconColor($clr-icon-color-error);
```

Comment:
Yes. I agree. I will look into my options to DRY the code. One them could be SASS `for` loop, I think.